### PR TITLE
fix: Prod bug for config api rendering

### PR DESCRIPTION
**Pull request by shivani170**
# Description

 Uanble to render data  for the list  of approvals via getConfig api.

Fixes

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx [hunk 1]
```diff
@@ -152,6 +152,7 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
             setPageViewType(ViewType.LOADING)
             setSelectedAppList([])
             getWorkflowsData()
+            getConfigs()
```
Comment (vivek-devtron) [inline src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx:155]: Why is this required? It was getting called on basis of url change

**Review by vivek-devtron — APPROVED** [review]

File: src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx (hunk 1 above)
Comment (shivani170) [reply src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx]: It was not calling due to handledLocation check. And so called in mounting
